bHY6yh-de1103/cod-pu6y7d_bHY6yh-de1103.xml

```diff
@@ -60,43 +60,43 @@
     </p>
             <p xml:id="bHY6yh-de1103-d1e119">
                <quote xml:id="bHY6yh-de1103-Qd1e121">
-                  <lb ed="#V" n="37"/>Iob hystoria 34. c. Opus ho
+                 <ref> <lb ed="#V" n="37"/>Iob hystoria 34. c. </ref> Opus ho
         <lb ed="#V" n="38"/>Eminis reddet ei deus: et
         <lb ed="#V" n="39"/>iuxta vias singulorum restituet. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e131">
                <lb ed="#V" n="40"/>
-               <quote xml:id="bHY6yh-de1103-Qd1e135">¶ In genesi. c. 4. Nonne si bene
+               <ref xml:id="bHY6yh-de1103-Qd1e135">¶ In genesi. c. 4. </ref> Nonne si bene
         <lb ed="#V" n="41"/>egeris recipies. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e142">
                <lb ed="#V" n="42"/>
-               <quote xml:id="bHY6yh-de1103-Qd1e146">¶ Psalmista. 61 psalmus. Deus reddet vnicuique
+               <ref xml:id="bHY6yh-de1103-Qd1e146">¶ Psalmista. 61 psalmus. </ref> Deus reddet vnicuique
         <lb ed="#V" n="43"/>iuxta opera sua. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e152"> /
-      <quote xml:id="bHY6yh-de1103-Qd1e154"> 149 psalmus. Dominus iustus conci
+      <ref xml:id="bHY6yh-de1103-Qd1e154"> 149 psalmus. </ref> Dominus iustus conci
         <lb ed="#V" n="44"/>det ceruices peccatorum. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e160">
                <lb ed="#V" n="45"/>
-               <quote xml:id="bHY6yh-de1103-Qd1e164">¶ Salomon in prouer. c. 16. Cum placuerint <!--26.xml-->
+               <ref xml:id="bHY6yh-de1103-Qd1e164">¶ Salomon in prouer. c. 16. </ref> Cum placuerint <!--26.xml-->
         <cb ed="#V" n="b"/>
                   <lb ed="#V" n="46"/>domino vie hominus inimicos quoque suos conuer
         <lb ed="#V" n="47" break="no"/>tet ad pacem. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e176"> /
-      <quote xml:id="bHY6yh-de1103-Qd1e178"> Idem in ecclesiaste. c. 9. Qui in
+      <ref xml:id="bHY6yh-de1103-Qd1e178"> Idem in ecclesiaste. c. 9. </ref> Qui in
         <lb ed="#V" n="48"/>vno peccauerit multa bona perdet. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e184">
                <lb ed="#V" n="49"/>
-               <quote xml:id="bHY6yh-de1103-Qd1e188">¶ Regum li. i. c. 26 Deus retribuet vnicuique
+               <ref xml:id="bHY6yh-de1103-Qd1e188">¶ Regum li. i. c. 26. </ref> Deus retribuet vnicuique
         <lb ed="#V" n="50"/>secundum iustitiam suam et fidem. </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e194">
                <lb ed="#V" n="51"/>
-               <quote xml:id="bHY6yh-de1103-Qd1e198">¶ hieremias propheta in li. prophetiarum. c. 32. </quote>
+               <ref xml:id="bHY6yh-de1103-Qd1e198">¶ hieremias propheta in li. prophetiarum. c. 32. </ref> </quote>
             </p>
             <p xml:id="bHY6yh-de1103-d1e203">
```

Comment:
Looks like we have a paragraph where the ref and quote have been separated. So we'll need to combine this. We look at this issue next monday when we meet